scripts/build_deb.py

```diff
@@ -0,0 +1,333 @@
+#! /bin/env python3
+import argparse
+import json
+import logging
+import shutil
+import subprocess
+import re
+
+from argparse import ArgumentParser
+from dataclasses import dataclass
+from pathlib import Path
+
+@dataclass
+class SilKitVersion:
+    major: int
+    minor: int
+    patch: int
+    suffix: str
+
+@dataclass
+class SilKitInfo:
+    silkit_source_url: str
+    silkit_source_ref: str
+    silkit_source_path: Path
+    is_local: bool
+    recursive: bool
+
+@dataclass
+class BuildInfo:
+
+    silkit_pkg_path: Path
+    silkit_info: SilKitInfo
+    version: SilKitVersion
+    platform: str
+    work_dir: Path
+    keep_temp: bool
+    output_dir: Path
+
+@dataclass
+class BuildFlags:
+    add_platform_flags: str
+    add_debuild_flags: str
+    c_compiler: str
+    cxx_compiler: str
+
+logger = logging.getLogger("build_deb")
+
+def create_arg_parser() -> ArgumentParser:
+    ap = ArgumentParser("BuildPackages")
+    ap.add_argument("--build-cfg", type=Path, required=True)
+    ap.add_argument("--verbose", "-v", action='store_true', required=False)
+
+    return ap
+
+def load_cfg(cfg_path: Path):
+
+    with open(cfg_path, 'r') as f:
+        obj = json.load(f)
+        logger.debug(f"Build config: \n{obj}")
+        return obj
+
+def cleanup(build_info: BuildInfo, exitCode: int):
```

Comment:
```suggestion
def cleanup_and_exit(build_info: BuildInfo, exitCode: int):
```
I would normally call a function which exits just `die()` to show that it will terminate

Reply:
sounds good. I will probably just separate it again into a `exit + die `  function